src/validations/admins.js

```diff
@@ -5,14 +5,14 @@ const adminValidation = (req, res, next) => {
     firstName: Joi.string()
       .min(3)
       .max(20)
-      .regex(/^[A-Z][a-z]+$/),
+      .regex(/^[a-zA-Z]+$/),
```

Comment:
Fede I think we could change this regex for one that accepts compound names, since for example it would not accept "juan manuel".